lib/SPIRV/libSPIRV/SPIRVValue.h

```diff
@@ -66,7 +68,6 @@ class SPIRVValue : public SPIRVEntry {
              SPIRVType *TheType)
       : SPIRVEntry(M, TheWordCount, TheOpCode), Type(TheType) {
     setHasNoId();
-    SPIRVValue::validate();
```

Comment:
Why this line was removed?

Reply:
I guess, because of incidental double "d".

Reply:
Fixed